android/autodispose-androidx-lifecycle/src/androidTest/java/autodispose2/androidx/lifecycle/AndroidLifecycleScopeProviderTest.java

```diff
@@ -43,11 +43,11 @@ public void observable_beforeCreate() {
     final PublishSubject<Integer> subject = PublishSubject.create();
 
     // Spin it up
-    TestLifecycleOwner lifecycle = TestLifecycleOwner.create();
+    TestLifecycleOwner lifecycle = new TestLifecycleOwner();
     subject.to(autoDisposable(AndroidLifecycleScopeProvider.from(lifecycle))).subscribe(o);
-    lifecycle.emit(Lifecycle.Event.ON_CREATE);
-    lifecycle.emit(Lifecycle.Event.ON_START);
-    lifecycle.emit(Lifecycle.Event.ON_RESUME);
+    lifecycle.handleLifecycleEvent(Lifecycle.Event.ON_CREATE);
+    lifecycle.handleLifecycleEvent(Lifecycle.Event.ON_START);
+    lifecycle.handleLifecycleEvent(Lifecycle.Event.ON_RESUME);
```

Comment:
Note that `handleLifecycleEvent(Lifecycle.Event.ON_RESUME)` will catch the Lifecycle up through all events, so there's no technical reason to call `handleLifecycleEvent` multiple times in a row like this.

You might also consider using `lifecycle.currentState = Lifecycle.State.RESUMED`, which does the exact same thing.

Reply:
neat!